bcdaworker/main.go

```diff
@@ -204,17 +204,28 @@ func writeBBDataToFile(bb client.APIClient, acoID string, beneficiaryIDs []strin
 
 	w := bufio.NewWriter(f)
 	errorCount := 0
-	totalBeneIDs := float64(len(beneficiaryIDs))
+	totalBeneIDs := float64(len(cclfBeneficiaryIDs))
 	failThreshold := getFailureThreshold()
-
-	for _, beneficiaryID := range beneficiaryIDs {
-		pData, err := bbFunc(beneficiaryID, jobID)
+	db := database.GetGORMDbConnection()
+	defer db.Close()
+	for _, cclfBeneficiaryID := range cclfBeneficiaryIDs {
+		var cclfBeneficiary models.CCLFBeneficiary
+		db.First(&cclfBeneficiary, cclfBeneficiaryID)
+		blueButtonID, err := cclfBeneficiary.GetBlueButtonID(bb)
+		if err != nil {
+			log.Error(err)
+			errorCount++
+			appendErrorToFile(acoID, responseutils.Exception, responseutils.BbErr, fmt.Sprintf("Error retrieving BlueButton ID for cclfBeneficiary %s", cclfBeneficiaryID), jobID)
+		}
+		cclfBeneficiary.BlueButtonID = blueButtonID
+		db.Save(&cclfBeneficiary)
```

Comment:
Technically, we are finding the BB ID in the DB each time in current unit test scenario.  To ensure we don't have any surprises downstream, and to ensure we are testing both scenarios (ID exists and ID doesn't exist), we should definitely deploy this to AWS before we merge.  
To do this, would we need to update the `dev` DB with data in the new tables?